src/verify.c

```diff
@@ -1117,3 +1114,21 @@ enum swupd_code verify_main(int argc, char **argv)
 	progress_finish_steps("verify", ret);
 	return ret;
 }
+
+enum swupd_code diagnose_main(int argc, char **argv)
+{
+	int ret = SWUPD_OK;
+
+	string_or_die(&cmdline_option_picky_tree, "/usr");
```

Comment:
the problem is that when the picky-tree option is being read from the config file we need to allocate memory to store the value or it would be lost. So that means we would need a pointer to a char to read that value instead of const char.